apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -558,7 +559,7 @@ private Task KeepAlive(CancellationToken cancellationToken)
                 Sql = "SELECT 1"
             };
 
-            var waitTime = (int) ConnectionPoolOptions.KeepAliveTimeSpan.TotalMilliseconds;
+            var waitTime = (int) ConnectionPoolOptions.KeepAliveInterval.TotalMilliseconds;
```

Comment:
I assume Task.Delay doesn't have an overload accepting TimeSpan in the version you're using?